### DISCUSSION TITLE
Equivalent Release to "Preview"

### DISCUSSION THREAD

**Discussion by VincentNewkirk** [getredash/redash#6750] [Q&A]
Hello! I previously had `redash:preview` tag deployed in my k8. A couple days ago, there was a push to that image tag which broke my redash app. Since then there have been a couple more pushes to that tag. I tried switching the tag to `redash:10.1.0.b50633`, but my app is still not working. After logging in I encounter a db error that a user profile column is not there. I tried to run the migrations, but that errors as well.

Is there a tag equivalent or close to what was being used for `preview` until it was recently updated?

Thank you

**Comment by justinclift** [comment]
> Is there a tag equivalent or close to what was being used for preview until it was recently updated?

Nope.

Instead of that, we should probably figure out what's broken now and get it fixed.

If you try the `redash:preview` tag that's on Docker Hub at the moment, what's the exact error you're getting?  (please cut-n-paste it or maybe screenshot it)

**Reply by VincentNewkirk** [reply]
I attached a screenshot in a different comment below.
I was actually thinking it's better to use a stable version instead in case another update happens to `preview` in the future

**Comment by VincentNewkirk** [comment]
Thanks for your reply. This is the error I get now. The login page won't appear and this errors with 500:
<img width="747" alt="image" src="https://github.com/getredash/redash/assets/18303749/3bda0619-784a-4521-b734-c894de8ffd79">

**Comment by eradman** [comment]
@VincentNewkirk 

Does running the schema migrations fix this? (also please include the output from this)

```
docker compose run --rm server manage db upgrade
```
